tests/requirements.txt

```diff
@@ -1,3 +1,4 @@
 coverage
 codecov
 parameterized
+dwave-system>=1.13.0
```

Comment:
I see this creates a conflict with main requirements file. We'll have to update it there instead.

Reply:
When I tested locally, I ran `pip install -r requirements.txt` and `pip install -r test/requirements.txt` sequentially.  I see that CircleCI (`.circleci/config.yml`) is doing `pip install -r requirements.txt -r test/requirements.txt` in one command.  So, alternatively, I suppose we could change `.circleci/config.yml` to run those commands sequentially.  Should I try that?

Reply:
✔️